admin_manual/maintenance/upgrade.rst

```diff
@@ -36,14 +36,17 @@ After the upgrade is complete re-enable any which are compatible with the new re
 Upgrade Options
 ---------------
 
-There are three ways to upgrade your ownCloud server:
+There are three ways to upgrade your ownCloud server.
+The first — *and recommended* — way is to perform a :doc:`manual upgrade <manual_upgrade>`, using `the latest ownCloud release <owncloud.org/install/>`_.
+The second way is to use your distribution's :doc:`package manager <package_upgrade>` in addition with our official ownCloud repositories. 
```

Comment:
"in addition to" or "in conjunction with"?
"in addition with" is not common English.

Reply:
Thanks for spotting that.